### PR TITLE
fix(linkedin): fix broken link

### PR DESCRIPTION
**Pull request by wtfluckey**
LinkedIn link was broken - needs the `https`

giphy-1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,7 +34,7 @@ <h1 class="title">
       </span>
     </a>
   
-    <a href="www.linkedin.com/in/alyfluckey" target="_blank">
+    <a href="https://www.linkedin.com/in/alyfluckey" target="_blank">
```
Comment (wtfluckey) [inline index.html:37]: giphy-2

Comment (leongersing) [reply index.html:37]: almost

**Comment by leongersing** [comment]
simply the best

**Review by leongersing — APPROVED** [review]
ready for prime time

**Pull request merged**